static/javascripts/components/survey_card.jsx

```diff
@@ -392,7 +422,13 @@ var CheckboxQuestion = React.createClass({
                       <button className="mdl-button mdl-js-button mdl-button--fab mdl-button--mini-fab" onClick={this.addOption}>
                         <i className="material-icons">add</i>
                       </button>
-                      &nbsp;&nbsp;&nbsp; ADD OPTION
+                      &nbsp;&nbsp;&nbsp; ADD OPTION &nbsp;&nbsp;&nbsp;
+                    </p>
+                    <p>
+                      <button className="mdl-button mdl-js-button mdl-button--fab mdl-button--mini-fab" onClick={this.removeOption}>
+                        <i className="material-icons">subtract</i>
```

Comment:
So @Bradyta I think you would know how to get a subtract (i.e. - ) icon here right?